src/review/logic.py

```diff
@@ -49,7 +50,8 @@ def get_reviewers(article, candidate_queryset, exclude_pks):
         ).exclude(date_complete__isnull=True).order_by("-date_complete")
     )
     active_reviews_count = models.ReviewAssignment.objects.filter(
-        is_complete=False,
+        Q(date_complete__isnull=True) | Q(is_complete=True),
```

Comment:
Did you mean to use an XOR operator? 
>[Q](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#django.db.models.Q) objects and querysets can now be combined using ^ as the exclusive or (XOR) operator. XOR is natively supported on MariaDB and MySQL. For databases that do not support XOR, the query will be converted to an equivalent using AND, OR, and NOT.

https://docs.djangoproject.com/en/4.2/releases/4.1/#models

Also, it would be good to use a comment here explaining the implicitness of the query (we are trying to consider  declined reviews) or maybe even use a variable/custom filter, e.g

`INCOMPLETE_AND_NOT_DECLINED_FILTER = (Q(date_complete__isnull=True)^ Q(is_complete=True))`